pages/docs/tutorials/getting-started/servers.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Servers"
+date: 2019-04-03T10:56:52+01:00
+menu:
+  docs:
+    parent: 'getting-started'
+weight: 110
+---
+
+In the previous lesson, you learned how to create the definition of a simple [Hello World application](/docs/getting-started/hello-world). Let's take it from there.
+
+In this article, you'll learn how to add `servers` to your AsyncAPI document. Adding and defining servers is useful, because it specifies where and how to connect. The connection facilitates where to send and receive messages.
```

Comment:
```suggestion
In this tutorial, you'll learn how to add `servers` to your AsyncAPI document. Adding and defining servers is useful because it specifies where and how to connect. The connection facilitates where to send and receive messages.
```